@xen-orchestra/web/src/components/backups/configuration/BackupJobGeneralInformation.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <UiCard class="backup-job-general-information">
+    <UiTitle>{{ t('general-information') }}</UiTitle>
+    <VtsColumns>
+      <VtsColumn>
+        <VtsQuickInfoRow :label="t('name')" :value="backupJob.name" />
+      </VtsColumn>
+      <VtsColumn>
+        <VtsQuickInfoRow :label="t('id')" :value="backupJob.id" />
+      </VtsColumn>
+      <VtsColumn>
+        <VtsQuickInfoRow :label="t('mode')">
+          <template v-if="modeLabels.length > 0" #value>
+            <UiTagsList>
+              <UiTag v-for="label in getModeLabels(backupJob)" :key="label" variant="secondary" accent="info">
+                {{ label }}
+              </UiTag>
+            </UiTagsList>
+          </template>
```

Comment:
```suggestion
          <template v-if="modeLabels.length > 0" #value>
            <UiTagsList>
              <UiTag v-for="label in modeLabels" :key="label" variant="secondary" accent="info">
                {{ label }}
              </UiTag>
            </UiTagsList>
          </template>
```
We already have modeLabs in the script